app/components/DeleteButton/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+*
+* DeleteButton
+*
+*/
+
+import React from 'react';
+
+import styles from './styles.css';
+
+function DeleteButton(props) {
+  return (
+    <button
+      className={styles.deleteButton}
+      onClick={() => {
+        const response = window.confirm(props.confirmText); // eslint-disable-line no-alert
```

Comment:
Only disabled on this line. Normally eslint gives a warning when using window.alert/window.prompt/window.alert